src/libNode/MicroBlockProcessing.cpp

```diff
@@ -1601,8 +1630,4 @@ bool Node::MicroBlockValidator(const vector<unsigned char>& microblock,
     }
 
     return valid;
```

Comment:
Just feel uncomfortable on this kind of checking, with a strange while(false). (I know it's not the modification of this PR). I think only one if condition is needed for the checking. If possible, could you please help to modify?

Reply:
Historically, this was because we would have each check in a separate if condition, and any time that condition fails, we would break out of the while loop. I can remove the while loop now.